### PR TITLE
1.3/develop

### PR DESCRIPTION
**Pull request by PascalLeMerrer**
Fixed some typos in docs.

### CODE REVIEW

File: general/unit_testing.html [hunk 1]
```diff
@@ -180,7 +180,7 @@ <h3 id="unit_tests_for_modules">
 
 
 			<h3 id="testing_in_a_renamed_directory">
-				<a class="internal_link" href="#testing_in_a_renamed_directory">Tesing In A Renamed fuel/ Directory</a>
+				<a class="internal_link" href="#testing_in_a_renamed_directory">Testing in a renamed Fuel directory</a>
```
Comment (kenjis) [inline general/unit_testing.html:183]: "fuel/ Directory" is more clear for me.

Comment (frankdejonge) [inline general/unit_testing.html:183]: the fuel directory is in lower case, so it should be named like that here too.

**Comment by frankdejonge** [comment]
Apart from the fuel directory, this will get merged, thanks!

**Comment by WanWizard** [comment]
This can not be merged anymore. If still relevant, please update the pull request.

**Pull request closed without merging**